app/fileMenu.js

```diff
@@ -1,17 +1,28 @@
+const EDITOR = require("./editor");
+const cheerio = require('cheerio');
+
 module.exports = {
   fileMenu,
   newFile,
+  handleOpen,
   handleNew,
   handleSave,
   handleSaveAs,
 };
-let saveFlag;
+let saveFlag, openFlag;
 const getSavePath = () => savePath;
 const setSavePath = path => {
   savePath = path;
 };
 
+const getOpenPath = () => openPath;
+const setOpenPath = path => {

```

Comment:
What is the reason for `setOpenPath`?
Shouldn't it just `setSavePath` the path from where the file is opened?

Reply:
i thought that we have to do that when we are dealing with opening and saving file. As previously did for saving.